Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -352,31 +378,43 @@ function createWrapper(wrapperId) {
             </td>
 
           </sec:authorize>
-          <td align="justify"><font size="2"> <a
+          <td align="justify"> <a
               id="${concept.entry.id}" data-toggle="modal"
               data-target="#detailsModal"
-              data-conceptid="${concept.entry.id}"><c:out
-                  value="${concept.entry.word}"></c:out></a></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.id}"></c:out></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.wordnetId}"></c:out></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.pos}"></c:out></font></td>
-          <td align="justify"><font size="2"><c:out
-                value="${concept.entry.conceptList}"></c:out></font></td>
+              data-conceptid="${concept.entry.id}" ><c:out
+                  value="${concept.entry.word}"></c:out></a></td>
+          <td class="entryID" align="justify" id="entryId" ><c:out
```

Comment:
ids need to be unique, but this is printed in a loop, right? some there will be several ids 'entryId'